app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "csv"
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    respond_to do |format|
+      format.html { render controller_path }
+      format.csv { render csv: produce_csv(usage_statistics(params[:start_date], params[:end_date])), filename: "data-export" }
+    end
+  end
+
+  def usage_statistics(start_date, end_date)
+    questions = I18n.t("coronavirus_form.groups").map { |_, v| v[:questions] }.reduce(:merge)
+
+    start_date = Date.parse(sanitize(start_date)).beginning_of_day
+    end_date = Date.parse(sanitize(end_date)).end_of_day
+
+    results = {}
+    questions.each_key do |question|
+      question_text = questions.dig(question, :title)
+      counts = FormResponse
+        .where(created_at: start_date..end_date)
+        .select(Arel.sql(%{created_at::date, form_response -> '#{question}', COUNT(*)}))
+        .group(Arel.sql(%{created_at::date, form_response -> '#{question}'}))
+        .pluck(Arel.sql(%{created_at::date, form_response -> '#{question}', COUNT(*)}))
+        .reject { |form_responses| form_responses[1].nil? }
```

Comment:
Thank you! `form_responses` is a good name.
Nitpick, I think `form_responses[1].nil?` could be written as `form_responses.second.nil?`.

Reply:
Sure, will change this